microservice-app/src/main/java/com/sdevelopment/skt/microservice/ServletInitializer.java

```diff
@@ -0,0 +1,13 @@
+package com.sdevelopment.skt.microservice;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
```

Comment:
Is this class required?

Reply:
It was generated by the spring boot starter. I'm not sure. I'll look into it.

Reply:
I found this in Stackoverflow:

Older Servlet containers don’t have support for the ServletContextInitializer bootstrap process used in Servlet 3.0. You can still use Spring and Spring Boot in these containers but you are going to need to add a web.xml to your application and configure it to load an ApplicationContext via a DispatcherServlet.

Inorder to create deployable war file is to provide a SpringBootServletInitializer subclass and override its configure method. This makes use of Spring Framework’s Servlet 3.0 support and allows you to configure your application when it’s launched by the servlet container. Typically, you update your application’s main class to extend SpringBootServletInitializer.

Reply:
In my case the starter generated an independent class that does that. According to that I need it since I'm going to deploy using a war file.

Reply:
Any other info source about this class?

Reply:
@yuninho2005 is this class intended to do something? If it's not the case, remove it